r/README.md

```diff
@@ -3,6 +3,7 @@
 <!-- badges: start -->
 
 [![cran](https://www.r-pkg.org/badges/version-last-release/arrow)](https://cran.r-project.org/package=arrow)
+<a href="https://apache.r-universe.dev"><img src="https://apache.r-universe.dev/badges/arrow" class="img-fluid" alt="R-universe status badge"></a>
```

Comment:
Just to check, is there a reason that this needs to be raw html unlike the other badges?

Reply:
No reason. I'll switch to markdown and also bump to the right of the R-CMD check badge, so that it is in the same position as the conda forge badge.